pkg/users/account-create-dialog.js

```diff
@@ -170,19 +172,42 @@ function suggest_username(realname) {
 export function account_create_dialog(accounts) {
     let dlg = null;
     const state = {
+        dialogLoading: true,
         real_name: "",
         user_name: "",
         password: "",
         password_confirm: "",
         locked: false,
         confirm_weak: false,
         change_passw_force: false,
+        base_home_dir: null,
+        shell: null,
     };
     let errors = { };
 
     let old_password = null;
     let user_name_dirty = false;
 
+    function get_defaults() {
+        return cockpit.spawn(["useradd", "-D"], { superuser: "require" })
+                .catch(() => console.warn("Could not get useradd defaults. 'useradd -D' failed"))
```

Comment:
Please add `err: "message"` option and show the exception in the log, to make this debuggable.